src/clojure/nrepl/transport.clj

```diff
@@ -123,6 +130,40 @@
             (.close in)
             (.close out))))))))
 
+(defn stringify-everything
```

Comment:
Helpers should be private.

Reply:
Btw, this affects only keywords, right?

Reply:
The last two comments will be impacted by the protocol story.